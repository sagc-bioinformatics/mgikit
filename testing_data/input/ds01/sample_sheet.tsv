sample_id	i7	i5	i7_rc	i5_rc	template
Sample01	ACGGGCAT	GGCTAGAG	0	0	i78:i58
Sample02	ACGCTAAT	GGCTAGAG	0	0	i78:i58
Sample03	ACGCCCAT	GGCCAGAG	0	0	i78:i58
Sample04	GGGTCGAT	GGGGACTA	0	0	i78:i58
