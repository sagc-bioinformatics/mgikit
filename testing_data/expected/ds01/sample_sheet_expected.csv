sample_id,i7,i5,job_number,template,i7_rc,i5_rc
Sample01,ACGGGCAT,GGCTAGAG,.,i78:i58,0,0
Sample02,ACGCTAAT,GGCTAGAG,.,i78:i58,0,0
Sample03,ACGCCCAT,GGCCAGAG,.,i78:i58,0,0
Sample04,GGGTCGAT,GGGGACTA,.,i78:i58,0,0
